tests/util_test.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+"""PyTest for :mod:`pykern.util`
+
+:copyright: Copyright (c) 2022 Bivio Software, Inc.  All Rights Reserved.
+:license: http://www.apache.org/licenses/LICENSE-2.0.html
+"""
+import pytest
+from pykern.pkcollections import PKDict
+
+
+def test_is_pure_text():
+    from pykern import util
+    from pykern import pkunit
+
+    a = "a".encode("utf-8")
+    for case in [
+        PKDict(
```

Comment:
There is a subtlety here with for loops in tests. When a for loop fails in a test, you don't get the case printed. In this case, that might be ok. However, there's a lot of code here anyway, because of the PKDict. If you have a lot of repeated code in a local context, there's almost no value in using named dicts. Someone reading it has to pass through the boiler plate. Rather, try to model on pkunit's expect on left and actual on right, and create a nested functions and default the common case (is_truncated=False) and don't encode unless you have to, e.g.

```py
def _false(value, is_truncated=False):
    punit.pkeq(False, util.is_pure_text(value, is_truncated=is_truncated))
def _true(value, is_truncated=False):
    punit.pkeq(True, util.is_pure_text(value, is_truncated=is_truncated))
_false(b"\0 one null causes failure in valid text")
_false(False, a + b"\xc2")
_false(False, bytes(range(1, 0x20))
etc.
```

What this will show you is that you are missing cases, e.g. is_truncated=True and expect=False. There may be others missing.